tests/data/errata/invalid.json

```diff
@@ -0,0 +1,16 @@
+{
+  "errata": {
+    "rhba": {
+      "id": 120142,
+      "revision": 2,
+      "fulladvisory": "RHBA-2023:5017-02",
+      "status": "SHIPPED_LIVE"
+    },
+    "rhsa": {
+      "id": 120143,
+      "revision": 2,
+      "fulladvisory": "RHBA-2023:5018-02",
+      "status": "SHIPPED_LIVE"
+    }
+  }
+}
```

Comment:
There are 2 errata here, one of type rhba, and one of type rhsa. We only expect to get one type here, since we only looked up one erratum id. 
I added a comment to the test.